src/main/java/org/traccar/database/OpenIdProvider.java

```diff
@@ -94,9 +94,15 @@ public OpenIdProvider(Config config, LoginService loginService) {
     }
 
     public URI createAuthUri() {
+        Scope scope = new Scope("openid", "profile", "email");
+
+        if (adminGroup != null) {
+            scope.add("groups");
```

Comment:
So does it fail if you request it where it's not supported?

Reply:
It does, with Google you get a list of valid scopes and a list of invalid ones (just groups in this case)

Reply:
Then the PR looks good to me. Are you still working on it?

Reply:
Yeah I'll leave it drafted until I finish off the docs if thats good with you, incase I need to add anything else in